great_expectations/cli/suite.py

```diff
@@ -184,23 +182,20 @@ def _process_suite_new_flags_and_prompt(
     else:
         interactive_mode, profile = _suite_new_mode_from_prompt(profile)
 
-    return {
-        "interactive_mode": interactive_mode,
-        "profile": profile,
-    }
+    return interactive_mode, profile
```

Comment:
@cdkini Just curious: why tuple instead of dictionary -- to me, dictionary tells more clearly what is being returned.  Thanks.

Reply:
The utility method is only ever used in the one place and its return objects are immediately consumed.

A dictionary provides additional context but isn't entirely necessary here in my opinion. If we were to carry that dictionary across multiple lines, I'd be inclined to agree.